### PR TITLE
Do not pass -mvn /usr/bin/mvn

### PR DESCRIPTION
**Pull request by jglick**
This was obsolete as of #194!

### CODE REVIEW

**Review by imonteroperez — APPROVED** [review]

**Comment by jglick** [comment]
So @imonteroperez do you plan to merge this or what?

**Comment by imonteroperez** [comment]
> So @imonteroperez do you plan to merge this or what?

I was just waiting for an additional review, but LGTM to merge it.

**Pull request merged**
